Content.Server/Body/Surgery/SurgerySystem.cs

```diff
@@ -0,0 +1,560 @@
+
+                    id = "surgery-prepare-cancel-self-outsider-popup";
+                    part.Body.Owner.PopupMessageOtherClients(Loc.GetString(id,
+                        ("user", surgeon.Owner),
+                        ("item", drapes.Owner),
+                        ("part", target)),
+                        except: part.Body.Owner);
+                }
+                else
+                {
+                    var id = "surgery-prepare-cancel-self-no-zone-surgeon-popup";
+                    surgeon.Owner.PopupMessage(surgeon.Owner, Loc.GetString(id,
+                        ("item", drapes.Owner)));
+
+                    id = "surgery-prepare-cancel-self-no-zone-outsider-popup";
+                    surgeon.Owner.PopupMessageOtherClients(Loc.GetString(id,
+                        ("user", surgeon.Owner),
+                        ("item", drapes.Owner)));
+                }
+            }
+            else
+            {
+                if (IsReceivingSurgeryOnPart(target, out var part, out var body))
+                {
+                    var id = "surgery-prepare-cancel-surgeon-popup";
+                    body.Owner.PopupMessage(surgeon.Owner, Loc.GetString(id,
+                        ("item", drapes.Owner),
+                        ("target", body.Owner),
+                        ("zone", part.Owner)));
+
+                    id = "surgery-prepare-cancel-target-popup";
+                    surgeon.Owner.PopupMessage(body.Owner, Loc.GetString(id,
+                        ("user", surgeon.Owner),
+                        ("item", drapes.Owner),
+                        ("zone", part.Owner)));
+
+                    id = "surgery-prepare-cancel-outsider-popup";
+                    surgeon.Owner.PopupMessageOtherClients(Loc.GetString(id,
+                        ("user", surgeon.Owner),
+                        ("item", drapes.Owner),
+                        ("target", body.Owner),
+                        ("zone", target)),
+                        except: body.Owner);
+                }
+                else
+                {
+                    var id = "surgery-prepare-cancel-no-zone-surgeon-popup";
+                    target.PopupMessage(surgeon.Owner, Loc.GetString(id,
+                        ("item", drapes.Owner),
+                        ("target", target)));
+
+                    id = "surgery-prepare-cancel-no-zone-target-popup";
+                    surgeon.Owner.PopupMessage(target, Loc.GetString(id,
+                        ("user", surgeon.Owner),
+                        ("item", drapes.Owner)));
+
+                    id = "surgery-prepare-cancel-no-zone-outsider-popup";
+                    surgeon.Owner.PopupMessageOtherClients(Loc.GetString(id,
+                        ("user", surgeon.Owner),
+                        ("item", drapes.Owner),
+                        ("target", target)),
+                        except: target);
+                }
+            }
+        }
```

Comment:
I removed drapes from a target and got this 
![image](https://user-images.githubusercontent.com/31366439/134106001-c089c23c-a822-412a-aa89-cb17a9f178e8.png)